bin/git-delete-submodule

```diff
@@ -34,13 +34,16 @@ git add '.gitmodules'
 [  "$(wc -l '.gitmodules' | cut -d' ' -f1)" = '0' ] && git rm -f '.gitmodules'
 
 # 3. Need to confirm and commit the changes for yourself
-if git submodule status >/dev/null 2>&1 \
-	&& ! git submodule status | grep "$NAME"; then
+git_status_text="$(git submodule status >/dev/null 2>&1)"
+git_status_exit=$?
+if [ "$git_status_exit" -eq 0 ] \
+	&& printf '%s' "$git_status_text" | grep "$NAME"; then
```

Comment:
Maybe you miss reverting the exit code in this commit?

Reply:
not sure what you mean, could you elaborate?

Reply:
@go2null 
If the operation is success, the `$NAME` should not be found in the `git submodule status` and `grep "$NAME"` should be failed.
If I am wrong, please correct me.

Reply:
you are correct, I changed to `grep -v`.